src/codegen/CodeGenerators.jl

```diff
@@ -88,6 +90,7 @@ All imported `.proto` files are compiled as well; an error is thrown if they can
 - `output_directory::Union{String,Nothing}=nothing`: Path to store the generated Julia source code. When omitted, the translated code is saved to temp directory, the path is shown as a @info log.
 
 # Keywords
+- `with_mutable_structs::Bool=false`: If set to `true`, generate structs that are mutable in nature. 
```

Comment:
Thanks! The overall idea looks good to me!

Let's shorten the name to `mutable_structs` and let's mention that this changes the _default_ struct "kind" -- in the future we might want to add more granularity for controlling this via comments etc.

I also think we should add the word `mutable` to the `JULIA_RESERVED_KEYWORDS` list in `src/codegen/names.jl`

```suggestion
- `mutable_structs::Bool=false`: If set to `true`, the generator will default to generating mutable structs. 
```